app/model/group.js

```diff
@@ -51,6 +51,10 @@ if ('production' === process.env.NODE_ENV) {
 // pre hooks used to sync with LDAP,
 // currently we only support to add, no modification
 groupSchema.pre('save', function (next) {
+    // disable ldap for dev
+  if (!process.env.NODE_ENV && !process.env.LDAP) {
```

Comment:
You have to actually check the value of NODE_ENV and this should be logical OR, **not** logical AND.